x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -8,7 +8,7 @@ export function createJestConfig({ kibanaDirectory, rootDir, xPackKibanaDirector
   const fileMockPath = `${kibanaDirectory}/src/dev/jest/mocks/file_mock.js`;
   return {
     rootDir,
-    roots: ['<rootDir>/plugins', '<rootDir>/legacy/plugins', '<rootDir>/legacy/server'],
+    roots: ['<rootDir>/plugins'],
```

Comment:
can we clean up `moduleNameMapper` as well? `uiExports`, `legacy/plugins/xpack_main`, etc.
The same for OSS jest config